cate/webapi/start.py

```diff
@@ -72,12 +73,21 @@
 
 
 # noinspection PyAbstractClass
-class WebAPIVersionHandler(WebAPIRequestHandler):
+class WebAPIInfoHandler(WebAPIRequestHandler):
     def get(self):
+        workspace_manager_mode = "unknown"
+        if self.application.workspace_manager:
```

Comment:
Check not needed: "self.application.workspace_manager is not None" is always true!